anyway/parsers/injured_around_schools_2022.py

```diff
@@ -0,0 +1,547 @@
+LOCATION_ACCURACY_PRECISE_INT = 1
+AGE_GROUPS = [2, 3, 4]
+INJURED_TYPE_PEDESTRIAN = 1
+INJURED_TYPES = [1, 6, 7]
+VEHICLE_TYPES = [15, 21, 23]
+CONTENT_ENCODING = "utf-8"
+HEBREW_ENCODING = "cp1255"
+ANYWAY_UI_FORMAT_MAP_ONLY = "https://www.anyway.co.il/?zoom=17&start_date={start_date}&end_date={end_date}&lat={latitude}&lon={longitude}&show_fatal=1&show_severe=1&show_light=1&approx={location_approx}&accurate={location_accurate}&show_markers=1&show_discussions=0&show_urban=3&show_intersection=3&show_lane=3&show_day=7&show_holiday=0&show_time=24&start_time=25&end_time=25&weather=0&road=0&separation=0&surface=0&acctype={acc_type}&controlmeasure=0&district=0&case_type=0&show_rsa=0&age_groups=1234&map_only=true"
+ANYWAY_UI_FORMAT_WITH_FILTERS = "https://www.anyway.co.il/?zoom=17&start_date={start_date}&end_date={end_date}&lat={latitude}&lon={longitude}&show_fatal=1&show_severe=1&show_light=1&approx={location_approx}&accurate={location_accurate}&show_markers=1&show_discussions=0&show_urban=3&show_intersection=3&show_lane=3&show_day=7&show_holiday=0&show_time=24&start_time=25&end_time=25&weather=0&road=0&separation=0&surface=0&acctype={acc_type}&controlmeasure=0&district=0&case_type=0&show_rsa=0&age_groups=1234"
+DATE_INPUT_FORMAT = "%d-%m-%Y"
+DATE_URL_FORMAT = "%Y-%m-%d"
+
+SCHOOLS_DATA_DIR = pathlib.Path(__file__).parent.parent.parent / 'static' / 'data' / 'schools'
+
+
+def get_bounding_box(latitude, longitude, distance_in_km):
+    latitude = math.radians(latitude)
+    longitude = math.radians(longitude)
+
+    radius = 6371
+    # Radius of the parallel at given latitude
+    parallel_radius = radius * math.cos(latitude)
+
+    lat_min = latitude - distance_in_km / radius
+    lat_max = latitude + distance_in_km / radius
+    lon_min = longitude - distance_in_km / parallel_radius
+    lon_max = longitude + distance_in_km / parallel_radius
+    rad2deg = math.degrees
+
+    return rad2deg(lat_min), rad2deg(lon_min), rad2deg(lat_max), rad2deg(lon_max)
+
+def acc_inv_query(longitude, latitude, distance, start_date, end_date, school):
+    lat_min, lon_min, lat_max, lon_max = get_bounding_box(latitude, longitude, distance)
+    baseX = lon_min
+    baseY = lat_min
+    distanceX = lon_max
+    distanceY = lat_max
+    pol_str = "POLYGON(({0} {1},{0} {3},{2} {3},{2} {1},{0} {1}))".format(
+        baseX, baseY, distanceX, distanceY
+    )
+
+    query_obj = (
+        db.session.query(InvolvedMarkerView)
+        .filter(InvolvedMarkerView.geom.intersects(pol_str))
+        .filter(
+            or_(
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_1_CODE),
+                (InvolvedMarkerView.provider_code == BE_CONST.CBS_ACCIDENT_TYPE_3_CODE),
+            )
+        )
+        .filter(InvolvedMarkerView.accident_timestamp >= start_date)
+        .filter(InvolvedMarkerView.accident_timestamp < end_date)
+        .filter(InvolvedMarkerView.location_accuracy == LOCATION_ACCURACY_PRECISE_INT)
```

Comment:
And then correct this line to `.filter(InvolvedMarkerView.location_accuracy.in_(LOCATION_ACCURACY_PRECISE_LIST))`